draft-ietf-ppm-dap.md

```diff
@@ -1786,21 +1808,23 @@ to the previous request, then the Helper MAY abort with error `invalidMessage`.
 (Note that a report may be missing, in which case the Helper should assume the
 Leader rejected it.)
 
-[OPEN ISSUE: Issue 438: It may be useful for the Leader to explicitly signal
-rejection.]
-
 Next, the Helper checks if the continuation step indicated by the request is
 correct. (For the first `AggregationJobContinueReq` the value should be `1`;
 for the second the value should be `2`; and so on.) If the Leader is one step
 behind (e.g., the Leader has resent the previous HTTP request), then the Helper
-MAY attempt to recover by re-sending the previous `AggregationJobResp`. In this
-case it SHOULD verify that the contents of the `AggregationJobContinueReq` are
+MAY attempt to recover by sending HTTP 202 status Accepted. In this case it
```

Comment:
What is the Helper trying to accomplish here? Does this response actually hep? If I understand correctly, the Helper's response would be the same had the Leader indicated the correct step.

Reply:
> if the Helper receives a repeated request it must treat it as if it needs to be handled asynchronously (since there is no way to communicate synchronous completion), though I suspect the implementation would effectively be to treat the job as already completed & return the results on the first attempt to poll.

That's more or less correct. My idea is that the repeated request is effectively a no-op, i.e. the helper-side aggregation job state does not need to change.

I've added verbiage
> If the Leader is one step behind (e.g., the Leader has resent the previous HTTP request), then the Helper MAY attempt to recover by sending HTTP 202 status Accepted and discontinuing any further work.

to cut the transition to the next paragraph. Let me know if more elaboration is necessary.